src/coreclr/jit/lsra.cpp

```diff
@@ -7374,9 +7374,12 @@ void LinearScan::insertSwap(
 // getTempRegForResolution: Get a free register to use for resolution code.
 //
 // Arguments:
-//    fromBlock - The "from" block on the edge being resolved.
-//    toBlock   - The "to" block on the edge
-//    type      - the type of register required
+//    fromBlock              - The "from" block on the edge being resolved.
+//    toBlock                - The "to" block on the edge. Can be null for shared critical edge resolution.
+//    type                   - The type of register required
+//    terminatorConsumedRegs - Registers consumed by 'fromBlock's terminating node.
```

Comment:
can you make the order of the arguments consistent with `resolveEdge()`? Have `sharedCriticalLiveSet` and then `terminatorConsumedRegs`?